src/js/model/primitives/Primitive.js

```diff
@@ -24,18 +30,48 @@ function clean(spec, resolve) {
       delete spec[k];
     }
     else if (dl.isObject(p)) {
-      spec[k] = p.signal && res ? sg.value(p.signal) : clean(spec[k], resolve);
+      spec[k] = p.signal && cln ? sg.value(p.signal) : _clean(spec[k], clean);
     }
   }
 
   return spec;
 }
 
-prototype.export = function(resolve) {
-  return clean(dl.duplicate(this), resolve);
+/**
+ * Initializes the Primitive. This is often where properties will be replaced
+ * with Lyra-specific signals to drive reactive updates.
+ * @return {Object} The Primitive.
+ */
+Primitive.prototype.init = function() {
+  return this;
 };
 
-prototype.parent = function(pid) {
+/**
+ * Exports the primitive as a complete Vega specification.
+ * @param  {boolean} [clean=true] - Should Lyra-specific properties be removed
+ * or resolved (e.g., converting property signal references to actual values).
+ * @return {Object} A Vega specification.
+ */
+Primitive.prototype.export = function(clean) {
+  return _clean(dl.duplicate(this), clean);
+};
+
+/**
+ * Exports the primitive as a complete Vega specification with extra definitions
+ * to power Lyra-specific interaction (e.g., extra manipulator mark definitions).
+ * @return {Object} A Vega specification.
```

Comment:
OK, that's a departure from how I'd interpreted it—given that we can specify a constructor in our `@param` definitions or [type specifications](http://usejsdoc.org/tags-type.html), I was expecting consistency through `@returns`. Put another way, nothing in the jsdoc docs I can find implies to me that this wouldn't be preferable:

```js
/**
 * @returns {MyClass}
 */
function returnClass() { return new MyClass(); }

/**
 * @param {MyClass} classInstance
 */
function doSomethingWithInstance( classInstance ) { classInstance.everything(); }
```